Specs/DateTime/Spanish/DatePeriodExtractor.json

```diff
@@ -4611,5 +4611,41 @@
         "Length": 14
       }
     ]
+  },
+  {
+    "Input": "semana pasada",
+    "NotSupportedByDesign": "javascript, python",
+    "Results": [
+      {
+        "Text": "semana pasada",
+        "Type": "daterange",
+        "Start": 0,
+        "Length": 13
+      }
+    ]
+  },
+  {
+    "Input": "la semana pasada",
+    "NotSupportedByDesign": "javascript, python",
```

Comment:
Could we enable these tests in js and python too? Or is there a complicating factor?